### PR TITLE
Add time-picker and date-picker input group template sample.

### PR DESCRIPTION
**Pull request by npaunov**

### CODE REVIEW

File: en/components/date_picker.md [hunk 1]
```diff
@@ -125,6 +125,42 @@ Here is the formatted date:
 </div>
 <div class="divider--half"></div>
 
+#### Templating Input Group
+
+The Date Picker's input group look can be customized. To do that we need to decorate a ng-template inside the date picker with `IgxDatePickerTemplate` directive and use the context returned to customize the way the date picker's input is displayed. You can make use of `value`, `displayData` and the `openDialog`.
```
Comment (bkulov) [inline en/components/date_picker.md]: See my comment for the time picker. Copy the modified texts into the JP versions as well.

File: en/components/date_picker.md [hunk 2]
```diff
@@ -125,6 +125,42 @@ Here is the formatted date:
...
+
+In the following example we modify the default label "Date" add a second icon as suffix and an additional label:
```
Comment (bkulov) [inline en/components/date_picker.md]: Replace `and an additional label` with something like `Bellow the input group we're using a label to display the real time picker value`.
The same applies to time picker as well.

**Comment by zdrawku** [comment]
@yoshimis I am adding label pending-localization to this PR

**Comment by bkulov** [comment]
@yoshimis Please create a new branch and PR for this as the original branch is deleted.

**Pull request merged**